app/helpers/value_groups_helper.rb

```diff
@@ -1,17 +1,29 @@
 module ValueGroupsHelper
   def value_group_pie_data(value_group)
-    value_group.children
+    values = value_group.children
       .map do |child|
         {
           label: to_accountable_title(Accountable.from_type(child.name)),
           percent_of_total: child.percent_of_total.round(1).to_f,
           value: child.sum.amount.to_f,
           currency: child.sum.currency.iso_code,
+          currency_symbol: child.sum.currency.symbol,
+          formatted_value: format_money(child.sum.amount.to_f, precision: 2, unit: child.sum.currency.symbol, separator: ".", delimiter: ","),
           bg_color: accountable_bg_class(child.name),
           fill_color: accountable_fill_class(child.name)
         }
       end
       .filter { |child| child[:value] > 0 }
-      .to_json
+
+    total = values.sum { |child| child[:value] }
+    formatted_total = format_money(total, precision: 2, unit: value_group.sum.currency.symbol, separator: ".", delimiter: ",")
```

Comment:
I think we're getting closer here, but we cannot rely on the `separator` and `delimiter` being fixed values.  These will be reversed in some cases depending on the currency that is being formatted.

A potential data structure that we could pass to the frontend could be:

```
{
  cents_str: child.sum.cents_str,
  dollar_str: format_money_without_symbol(child.sum, precision: 0)
  currency_symbol: child.sum.currency.symbol
}
```

This allows us to retain the original value while also getting the individual parts of the value that can be independently displayed.